python/cudf/utils/cudautils.py

```diff
@@ -727,3 +727,15 @@ def row_matrix(cols, nrow, ncol, dtype):
                                                       col.to_gpu_array(),
                                                       nrow, ncol)
     return matrix
+
+
+@cuda.jit
+def gpu_modulo(arr, d):
+    i = cuda.grid(1)
+    if i < arr.size:
+        arr[i] %= d
+
+
+def modulo(arr, d):
```

Comment:
I wouldn't expect this and the corresponding kernel to always run inplace, I think you should have an input array and an output array, along with the scalar.